### DISCUSSION TITLE
AI、VR、AR和區塊鏈 將驅動元宇宙發展

### DISCUSSION THREAD

**Discussion by chun1203** [1101-GS4719-C/Python-Community-GS4719-C#327] [科技新知]
人工智慧(AI)、虛擬實境(VR)、擴增實境(AR)和區塊鏈(blockchain)等新興科技，將可望整合在一起，驅動元宇宙技術發展。VentureBeat網站認為，時下最夯的元宇宙概念蘊藏龐大成長空間，目前流行的科技和元宇宙差異在於後者具備的**沉浸式技術**，這也是Meta、微軟(Microsoft)、NVIDIA等業者重點投資的部分，這也可能是下一世代的網路技術。
[https://www.digitimes.com.tw/iot/article.asp?cat=158&id=0000626522_IGK6TTL81RML803AQRZ1K](url)

**Comment by 107602505** [comment]
感覺這些新興的科技勢必要整合在一起才能將各項的技術做最大的利用，藉由驅動元宇宙的這些技術慢慢改變人類的生活品質，或許未來大家就會像電影一級玩家中的人物一樣生活在虛擬的世界中。
